lib/rom/setup.rb

```diff
@@ -54,8 +33,8 @@ def schema(&block)
     #   end
     #
     # @api public
-    def relation(name, &block)
-      @relations.update(name => block)
+    def relation(name, options = {}, &block)
+      @relations.update(name => [options, block])
```

Comment:
Defining the same relation twice is not supported and it should blow up. I'll improve it in a separate PR